deno/gateway/v10.ts

```diff
@@ -2051,6 +2055,45 @@ export interface GatewayPresenceUpdateData {
  */
 export type GatewayActivityUpdateData = Pick<GatewayActivity, 'name' | 'state' | 'type' | 'url'>;
 
+export interface GatewayMessagePollVoteData {
```

Comment:
This is in the wrong region, this should be moved in the received payloads
Also needs to follow the name format (and ideally export aliases for the event name and dispatch data